packages/dashmate/src/listr/tasks/setup/nodeTypes.js

```diff
@@ -0,0 +1,32 @@
+const {
+  NODE_TYPE_MASTERNODE,
+  NODE_TYPE_FULLNODE,
+} = require('../../../constants');
+
+const NODE_TYPE_NAMES = {
+  MASTERNODE: 'masternode',
+  HP_MASTERNODE: 'high-performance masternode',
+  FULLNODE: 'fullnode',
+  HP_FULLNODE: 'high-performance fullnode',
```

Comment:
This type doesn't make sense to me. Full nodes have no collateral requirements and won't run platform services, right?

Reply:
Nevermind, saw the description elsewhere